src/InverseSolve/iteratedTikhonov.jl

```diff
@@ -0,0 +1,51 @@
+export iteratedTikhonov
+
+function iteratedTikhonov(m0,pInv::InverseParam,PF::Array{RemoteRef{Channel{Any}}},nBeta,betaFac,targetMisfit;indCredit=[],dumpResults::Function = dummy)
```

Comment:
please provide some documentation for the function so that
```
?iteratedTikhonov
```
works. Look, for example, at ```projGNCG```.